### ISSUE TITLE
uninitialized constant FPM::Package::Deb::Zlib (NameError)

### ISSUE THREAD

**Issue by jean-christophe-manciot** [jordansissel/fpm#1746]
Debian bullseye
fpm 1.11.0
ruby 1:2.7+2
zlib1g-dev 1:1.2.11.dfsg-2

Building a deb package leads to:
```
...
Converting dir to deb {:level=>:info}
Traceback (most recent call last):
	9: from /usr/local/bin/fpm:23:in `<main>'
	8: from /usr/local/bin/fpm:23:in `load'
	7: from /var/lib/gems/2.7.0/gems/fpm-1.11.0/bin/fpm:7:in `<top (required)>'
	6: from /var/lib/gems/2.7.0/gems/clamp-1.0.1/lib/clamp/command.rb:133:in `run'
	5: from /var/lib/gems/2.7.0/gems/fpm-1.11.0/lib/fpm/command.rb:574:in `run'
	4: from /var/lib/gems/2.7.0/gems/clamp-1.0.1/lib/clamp/command.rb:68:in `run'
	3: from /var/lib/gems/2.7.0/gems/fpm-1.11.0/lib/fpm/command.rb:487:in `execute'
	2: from /var/lib/gems/2.7.0/gems/fpm-1.11.0/lib/fpm/package/deb.rb:494:in `output'
	1: from /var/lib/gems/2.7.0/gems/fpm-1.11.0/lib/fpm/package/deb.rb:494:in `tap'
/var/lib/gems/2.7.0/gems/fpm-1.11.0/lib/fpm/package/deb.rb:495:in `block in output': uninitialized constant FPM::Package::Deb::Zlib (NameError)
```

Same issue when installing fpm with ruby2.3 (by changing the symlink /usr/bin/ruby --> ruby2.3 instead of ruby2.7) then trying to build the package.

**Comment by jean-christophe-manciot** [comment]
Same issue with fpm 1.10.2 & ruby2.7.

**Comment by jervi** [comment]
Duplicate of #1739

**Comment by jordansissel** [comment]
Fixed in FPM 1.12.0
